### PR TITLE
Fix variants tests in datagrid

### PR DESCRIPTION
**Pull request by eCzerniak**
I want to merge this change because it fixes tests for variants. 

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://marketplace-gray.vercel.app/

### Do you want to run more stable tests?
Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation

CONTAINERS=1

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6661 Datagrid](https://app.clickup.com/t/2549495/SALEOR-6661)

File: cypress/e2e/products/productsVariants.js [hunk 1]
```diff
@@ -79,32 +79,52 @@ describe("As an admin I should be able to create variant", () => {
...
+            .get(BUTTON_SELECTORS.showMoreButton)
+            .click()
+            .get(PRODUCT_DETAILS.editVariant)
+            .click()
+            .get(VARIANTS_SELECTORS.manageChannels)
+            .click()
+            .get(VARIANTS_SELECTORS.allChannels)
+            .check()
+            .get(BUTTON_SELECTORS.submit)
+            .click();
```
Comment (karola312) [inline cypress/e2e/products/productsVariants.js:109]: Can we split it to few functions? For eg. addVariantToDatagrid, changeVarinatNameInDatagrid, enterVariantEditPage, selectChannelsForVariant?

File: cypress/e2e/products/productsVariants.js [hunk 2]
```diff
@@ -130,26 +150,34 @@ describe("As an admin I should be able to create variant", () => {
...
-        .then(([firstVariant, secondVariant]) => {
-          expect(firstVariant).to.have.property("price", variants[0].price);
-          expect(secondVariant).to.have.property("name", variants[1].name);
-          expect(secondVariant).to.have.property("price", variants[1].price);
+
+          cy.visit(`${urlList.products}${createdProduct.id}`)
+            .get(BUTTON_SELECTORS.showMoreButton)
+            .click()
+            .get(PRODUCT_DETAILS.editVariant)
+            .click()
```
Comment (karola312) [inline cypress/e2e/products/productsVariants.js:158]: And here we could reuse enterVariantEditPage() function

File: cypress/e2e/products/productsVariants.js (hunk 1 above)
Comment (eCzerniak) [reply cypress/e2e/products/productsVariants.js:109]: If it is not a problem we could do this in another PR, ok? :)

**Review by karola312 — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**